src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CommencementDateController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Mvc;
+using NHSD.BuyingCatalogue.Ordering.Api.Models;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.Controllers
+{
+    [Route("api/v1/orders/{orderId}/commencement-date")]
+    [ApiController]
+    [Produces("application/json")]
+    public sealed class CommencementDateController : Controller
+    {
+        [HttpGet]
+        public ActionResult Get()
+        {
+            var result = new CommencementDateModel { CommencementDate = DateTime.UtcNow };
+
+            return Ok(result);
+        }
+
+        [HttpPut]
+        public ActionResult Update([FromRoute] [Required] string orderId,
+            [FromBody] [Required] CommencementDateModel model)

```

Comment:
The parameter attributes aren't needed and can be safely removed.

Reply:
👍